.github/workflows/ci-pipeline.yml

```diff
@@ -158,25 +158,74 @@ jobs:
         env:
           CHART: ${{ needs.get-chart.outputs.chart }}
         run: |
-          APP_VERSION="$(yq e '.appVersion' bitnami/${CHART}/Chart.yaml)"
           # Updating CRDs stored at 'bitnami/$CHART/crds' (not 'bitnami/$CHART/templates/crds')
-          mapfile -t crd_files < <(find "bitnami/${CHART}/crds" -name "*.yaml" -o -name "*.yml" 2>/dev/null || true)
+          mapfile -t crd_files < <(find "bitnami/${CHART}/crds" "bitnami/${CHART}/templates/crds" -name "*.yaml" -o -name "*.yml" 2>/dev/null || true)
           for file in "${crd_files[@]}"; do
             # Automatically update CRDs that use the '# Source' header
             source_url_tpl="$(head -n 1 $file | grep -E "^# ?Source: ?" | sed -E 's|^# ?Source: ?||' || true)"
             if [[ -n "$source_url_tpl" ]]; then
-              # Replace version placeholder, if present
-              source_url=$(echo "$source_url_tpl" | sed "s/{version}/${APP_VERSION}/")
               # Validate the second line of the CRD file includes the version of the CRD
               crd_version="$(head -n 2 $file | tail -n 1 | grep -E "^# ?Version: ?" | sed -E 's|^# ?Version: ?||' || true)"
               if [[ -z "$crd_version" ]]; then
                 echo "error=CRD file '${file}' does not include the '#Version: <version> header'"
                 exit 1
               fi
+              # Additional headers may be used for extra features
+              # Conditional - Adds a conditional {{if}}/{{end}} to the downloaded upstream CRD
+              # VersionOf - Name of a subcomponent, its version will be used for CRD tracking instead of the main component version
+              # UseKustomize - If set to true, uses Kustomize to render the CRDs
+              # SplitFile - If set to true, uses yq to filter resources having 'kind: CustomResourceDefinition', useful when using 'install.yaml' file as upstream source
```

Comment:
Not sure if `SplitFile` is the best name here. As I can  see the code is filtering a yml file to select only the CRDs and the result is just one file... Maybe `FilterFile` is a better option.

Reply:
Hmm, but the result won't be a single file either, `SplitFile` may yield one or more CRD files when an 'install.yaml' is provided.